docs-developer/deploying.md

```diff
@@ -28,11 +28,14 @@ if you have access.
 
 ## How to revert to a previous version
 
-In case the deployed version is bad we can revert to a previous version using
-Netlify's dashboard.
-
-[Search for the previous production build](https://app.netlify.com/sites/perf-html/deploys?filter=production)
-and deploy it from its detail page.
+The easiest way is to reset the production branch to a previous version, and
+force push it. You'll need to enable force-pushing for the branch production,
+using the [Branch Settings on GitHub](https://github.com/firefox-devtools/profiler/settings/branches).
```

Comment:
I think this would be good to add as a step in the deployment script, for example:

> Go to https://github.com/firefox-devtools/profiler/settings/branches and enable force pushing.
> Hit enter when you are read.

... Do the force push

> Please re-enable the force push protection
> https://github.com/firefox-devtools/profiler/settings/branches

Unless we are planning on turning this off everywhere.

Reply:
I added these steps, each one waiting for an `<enter>`